libs/api/bpf_syscall.cpp

```diff
@@ -3,96 +3,157 @@
 #include "bpf.h"
 #include "libbpf.h"
 
+#include <windows.h>
+#include <WinError.h>
```

Comment:
list includes in alphabetical order unless you get build errors.  If order does matter, add a // comment saying so.

Reply:
This sorting is enforced by clang-format as far as I can tell, which gives the windows headers higher priority.